eq-author-api/schema/tests/comments.test.js

```diff
@@ -215,4 +218,198 @@ describe("comments", () => {
       comments: expect.any(Array),
     });
   });
+
+  //////////////////////////////////////////Replies///////////////////////////////////
```

Comment:
You can nest describe blocks inside each other, so I would add another one in here instead of using a comment.

Reply:
aaah, but i like my comments :)